site/pages/renderers/__samples__/wrap-react.js

```diff
@@ -0,0 +1,21 @@
+// @jsx React.createElement
+
+import { props, withComponent } from 'skatejs';
+import { wrap } from '@skatejs/renderer-react';
+import React from 'react';
+
+class HelloReact extends React.Component {
+  render() {
+    return <span>Hello, {this.props.name}!</span>;
+  }
+}
+
+class WrapReact extends withComponent(wrap(HelloReact)) {
+  static get props() {
```

Comment:
This is more a discussion point / note for myself:

We could have another example that shows sharing Flow types for both components and you wouldn't even have to have a `props` getter here.

Reply:
👍 sounds good - would be happy to do, though I have no experience with Flow, so until I've gotten some experience wouldn't be able to contribute on this